### PR TITLE
fix(devtools): RHIF-6 - Make example .env defaults file

### PR DESCRIPTION
**Pull request by bastilian**
With this the `npm run start:proxy` and `npm run start:proxy:beta` should work out of the box.

## Secure Coding Practices Checklist GitHub Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [ ] Input Validation
- [ ] Output Encoding
- [ ] Authentication and Password Management
- [ ] Session Management
- [ ] Access Control
- [ ] Cryptographic Practices
- [ ] Error Handling and Logging
- [ ] Data Protection
- [ ] Communication Security
- [ ] System Configuration
- [ ] Database Security
- [ ] File Management
- [ ] Memory Management
- [x] General Coding Practices

### CODE REVIEW

File: config/dev.webpack.config.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /* eslint-disable camelcase */
-require('dotenv').config();
+require('dotenv').config({ path: '.env.example' });
```
Comment (vkrizan) [inline config/dev.webpack.config.js:2]: It's not convenient to auto-include example file. Maybe a rename to `.env.default` could be done?

Comment (bastilian) [reply config/dev.webpack.config.js:2]: Why is it not convenient?

Comment (bastilian) [reply config/dev.webpack.config.js:2]: It doesn't really matter what the names file is, honestly.

Comment (vkrizan) [reply config/dev.webpack.config.js:2]: Examples are meant to be copied over and changed before use. This changes them to be used always and/or by default. Well, naming convention does matter, as anyone can see the purpose of the file right from its name.

File: README.md [hunk 2]
```diff
@@ -29,7 +29,7 @@ $ npm run start:proxy:beta # Will run the UI with beta chrome
 To run the container setup either Podman or Docker and their compose commands can be used to
 
 ```shell
-  $ cp .env.example .env # Can be used to enable a local backend and other services
+  $ cp .env.defaults .env # Can be used to enable a local backend and other services
```
Comment (vkrizan) [inline README.md:32]: This step is redundant since the defaults is being loaded.

**Review by vkrizan — APPROVED** [review]
:heavy_check_mark:

**Pull request merged**